libs/cdk/utils/drag-and-drop/dnd-list/dnd-list.directive.ts

```diff
@@ -149,21 +153,21 @@ export class DndListDirective<T> implements AfterContentInit, OnDestroy {
     ngAfterContentInit(): void {
         this._changeDraggableState(this._draggable);
         this.dndItems.changes
-            .pipe(startWith(null), takeUntil(this._onDestroy$))
+            .pipe(startWith(null), takeUntilDestroyed(this._onDestroy$))
             .subscribe(() => this.refreshQueryList());
     }
 
     /** @hidden */
     ngOnDestroy(): void {
-        this._onDestroy$.next();
-        this._onDestroy$.complete();
+        this._refresh$.next();
+        this._refresh$.complete();
     }
 
     /**
      * Refreshes the indexes of the items.
      */
     refreshQueryList(): void {
-        const refresh$ = merge(this._refresh$, this._onDestroy$);
+        const refresh$ = merge(this._refresh$, destroyObservable(this._onDestroy$));
```

Comment:
Yeah, I think I messed up with `destroyObservable` function, takeUntilDestroyed(this._destroyRef)` does the same exact thing, but less maintenance from our end